app/models/match.rb

```diff
@@ -5,4 +5,6 @@ class Match < ApplicationRecord
 
     #belongs_to :user
     #uncomment when the user table is created for device
+    has_many :chessPieces
```

Comment:
This doesnt seem right, have you tested you code your code that you can make a chess piece for a match?